tests/e2e/standalone/publish_test.go

```diff
@@ -1,5 +1,5 @@
-//go:build e2e
-// +build e2e
+//go:build e2e && !template
+// +build e2e,!template
```

Comment:
These changes in all the files are for allowing template PRs alone be able to run locally.
In CI all PRs are run since only e2e tag is passed.